### DISCUSSION TITLE
Search Metadata

### DISCUSSION THREAD

**Discussion by tobias-hahn** [doccano/doccano#1616] [Q&A]
Hi,
I am working on a sequence labeling project and I was wondering if it is possible to use the search function to filter the dataset by metadata. That would be super helpful.
Thanks for your replies.

**Comment by strobetm** [comment]
Hello, also interested both in searching by metadata and text (within document and global search within project).
Virtual viewport, which renders to svg only currently visible text makes even browser "search on page" useless.
